.eslintrc

```diff
@@ -1,19 +1,75 @@
 {
-  "extends": "./node_modules/fbjs-scripts/eslint/.eslintrc",
+  "root": true,
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+  "extends": [
+    "airbnb",
+    "prettier",
+    "prettier/react"
+  ],
+  "plugins": [
+    "react",
+    "prettier"
+  ],
   "rules": {
-    // These are functional errors that could either be fixed or be locally disabled in specific
-    // files
-    "no-bitwise": 0,
-    "constructor-super": 0,
-    "no-this-before-super": 0,
-    "no-self-compare": 0,
-    "operator-assignment": 0,
-    "consistent-return": 0,
-
-    // These are stylistic errors that could be easily fixed
-    "semi": 0,
-    "comma-dangle": 0,
-    "space-before-function-paren": 0,
-    "curly": 0,
+    "no-empty": "warn",
```

Comment:
I marked with `warn` rules that preferred to be `error`. 
Most of them can be fixed with `--fix` flag.